partner_sale_pivot/views/sale_report_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_order_product_search" model="ir.ui.view">
+        <field name="name">sale.report.search.inherit</field>
+        <field name="model">sale.report</field>
+        <field name="inherit_id" ref="sale.view_order_product_search"/>
+        <field name="arch" type="xml">
+                <filter name="year" position="after">
+                    <filter string="Confirmed in" name="confirm_date_last_365_days"
+                            date="date" default_period="last_365_days"/>
```

Comment:
```suggestion
                    <filter string="Date" name="date_last_365_days"
                            date="date" default_period="last_365_days"/>
```
Here https://github.com/OCA/sale-workflow/pull/1064/files?file-filters%5B%5D=.png&file-filters%5B%5D=.py&file-filters%5B%5D=.rst&file-filters%5B%5D=.xml#diff-6e30398aa962d97acf2102fd5d1047d6R35 you refer to the filter called ```date_last_365_days```

Reply:
The string must be 'Date' because the 'date' field does not represent a confirmation date, what represents a confirmation date now in v13 is the 'date_order' field and this is not in the report.